app/models/logic/champ_value.rb

```diff
@@ -25,6 +25,7 @@ def compute(champs)
     targeted_champ = champ(champs)
 
     return nil if !targeted_champ.visible?
+    return nil if targeted_champ.value.blank?
```

Comment:
```suggestion
    return nil if targeted_champ.blank?
```

Je me demande si c'est pas plus safe de faire ça